src/style.css

```diff
@@ -1,3 +1,87 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+
+  --link-color: rgba(0, 172, 220, 1);
+}
+
+.header {
+  max-width: 1200px;
+  min-width: 1024px;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 1);
```

Comment:
no needs to use `rgba` since you don't add alpha-chanel. Either use `rgb` or `hex`

Reply:
<img width="597" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/f9855af3-e0a9-436f-b7bd-1b8958a40ee8">

Reply:
corrected